dnd.sty

```diff
@@ -9,9 +9,9 @@
 % Prerequisite Packages
 %
 
-% Loaded before tikz to avoid package option conflict with pgf.
-\RequirePackage[table]{xcolor}
+\PassOptionsToPackage{table}{xcolor} % Helps prevent option clash with other packages that include xcolor
 
+\RequirePackage{xcolor}
```

Comment:
With `\PassOptionsToPackage` above, can we alphabetize this import?

Reply:
Looks like it works, I will need to test it in one of my print builds to make sure.